lib/src/base-styles/_nx-lists.scss

```diff
@@ -7,7 +7,9 @@
 
 .nx-list {
   @include container-vertical;
-  margin: $nx-spacing-sm 0 $nx-spacing-md 0;
+
+  color: $nx-text-color-light;
+  margin: $nx-flow-content-spacing 0;
```

Comment:
Zeplin shows 16px spacing between the list and the previous header.  This impl has 32px

Reply:
> Zeplin shows 16px spacing between the list and the previous header. This impl has 32px

This is picking up the flow spacing which is now 24px. The standard bottom margin on an `h3` is set at 16px but I don't know if that is an old value or a new value. I want us to avoid having custom headers for components, we should just use standard headings. @dturner32 can you please add some insight into the `h3` bottom margin?

Reply:
> > Zeplin shows 16px spacing between the list and the previous header. This impl has 32px
> 
> This is picking up the flow spacing which is now 24px. The standard bottom margin on an `h3` is set at 16px but I don't know if that is an old value or a new value. I want us to avoid having custom headers for components, we should just use standard headings. @dturner32 can you please add some insight into the `h3` bottom margin?

This is the solution which I prefer: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/025892b3851164b80bbbb1bffda54e9dd60d0b9a

The "flow spacing" is only ever applied to the bottom, the top has a minimal (or no) setting which will adapt to the bottom margin of the object above. So in the case of an `h3` above we get the `h3`'s bottom margin of 16px, but if an `h2` is above we get the `h2`'s bottom margin of 24px.